gen/aircraft.py

```diff
@@ -1188,6 +1204,10 @@ def configure_behavior(
             group.points[0].tasks.append(OptRestrictJettison(restrict_jettison))
         if rtb_winchester is not None:
             group.points[0].tasks.append(OptRTBOnOutOfAmmo(rtb_winchester))
+        if do_aewc is not None:
+            group.points[0].tasks.append(AWACSTaskAction())
+        if freq is not None:
+            group.points[0].tasks.append(SetFrequencyCommand(freq.mhz))
```

Comment:
Raise these up to `configure_awacs`? I don't think they're valid for any other type.

We also can't use `SetFrequencyCommand` without breaking vaicom. `AircraftConflictGenerator._setup_group()` already sets up the frequency in a way that won't break vaicom, but that needs to be tweaked a bit to handle support units now.

It currently has:

```
channel = self.get_intra_flight_channel(unit_type)
group.set_frequency(channel.mhz)
```

That should become:

```
if flight.flight_type == FlightType.AEWC:
    channel = self.radio_registry.alloc_uhf()
else:
    channel = self.get_intra_flight_channel(unit_type)
group.set_frequency(channel.mhz)
```

That way it keeps vaicom working, and the `FlightData` that gets created also is correct (and we don't waste a channel on the infra-flight range, which is usually more constrained, although it doesn't matter here because they're all AI aircraft with no real restrictions).